docker/otel-collector/config.yaml

```diff
@@ -75,9 +85,8 @@ extensions:
     endpoint: :13133
 service:
   telemetry:
-    # TODO: enable internal metrics
-    # metrics:
-    #   address: ':8888'
+    metrics:
```

Comment:
can ticket this for later but seems better if we push directly? https://opentelemetry.io/docs/collector/internal-telemetry/#configure-internal-metrics

Reply:
oh interesting, I think it's fine to keep the pull-based setup, allowing users to export metrics